modules/python/src2/cv2_convert.cpp

```diff
@@ -701,6 +701,18 @@ bool pyopencv_to(PyObject* obj, String &value, const ArgInfo& info)
         return true;
     }
     std::string str;
+
+#if ((PY_VERSION_HEX >= 0x03060000) && !defined(Py_LIMITED_API)) || (Py_LIMITED_API >= 0x03060000)
```

Comment:
Need to update this check, because compilation fails in this configuration:

```
--   Python 3:
--     Interpreter:                 C:/utils/soft/python35-x64/python.exe (ver 3.5.4)
--     Libraries:                   C:/utils/soft/python35-x64/Libs/python35.lib (ver 3.5.4)
--     Limited API:                 YES (ver 0x03060000)
--     numpy:                       C:/utils/soft/python35-x64/lib/site-packages/numpy/core/include (ver 1.12.1)
--     install path:                python/cv2/python-3
```

https://pullrequest.opencv.org/buildbot/builders/precommit_opencl/builds/100307